### PR TITLE
pt fares

### LINKED ISSUE #961: pt fares

### PR DESCRIPTION
**Pull request by zishanbilal**
Fixed #961 

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/962)

### CODE REVIEW

**Comment by zishanbilal** [comment]
test!

**Comment by zishanbilal** [comment]
test!

File: src/main/resources/beam-template.conf [hunk 1]
```diff
@@ -31,6 +31,7 @@ beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.walk_intercept = "do
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.bike_intercept = "double | 0.0"
 beam.agentsim.agents.modalBehaviors.lccm.paramFile = ${beam.inputDirectory}"/lccm-long.csv"
 beam.agentsim.agents.modeSubsidy.file = ${beam.inputDirectory}"/subsidies.csv"
+beam.agentsim.agents.ptFare.file = ${beam.inputDirectory}"/ptFares.csv"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: please make sure to log error if file set in config, but not present.

File: src/test/scala/beam/router/TollRoutingSpec.scala [hunk 2]
```diff
@@ -100,16 +106,13 @@ class TollRoutingSpec
             Modes.BeamMode.CAR,
             asDriver = true
           )
-        ),
-        Access,
-        false,
-        timeValueOfMoney
+        )
```
Comment (michaz) [inline src/test/scala/beam/router/TollRoutingSpec.scala]: @zishanbilal Errr, I guess this was an oversight, but this test was actually testing something. Could you please revert those assertions and restore that functionality?

Comment (michaz) [reply src/test/scala/beam/router/TollRoutingSpec.scala]: (Kind of urgent, since we would like to merge a cost-related PR..)

**Pull request merged**